tests/test_disk_usage.py

```diff
@@ -0,0 +1,19 @@
+import pytest
```

Comment:
After the previous refactor, this file would be only testing the internals of `shutil.disk_usage`, which are tested by Python itself. If you want to create an acceptance test this must be inside [*tests/test_download_release.py*](https://github.com/mondeja/blender-downloader/blob/master/tests/test_download_release.py) and done mocking the download and `shutil.disk_usage` return value, but IMHO, I think that this little change does not need testing.

Reply:
It makes sense. I got carried away and overengineered low hanging fruit stuff, my bad.